modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1180,6 +1190,34 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private void _enableOracleSyntax() {
+		Props props = new PropsImpl();
+
+		String jdbcURL = props.get("jdbc.default.url");
+
+		String className = props.get("jdbc.default.driverClassName");
+
+		if (className.equals(JDBCDriver.class.getName())) {
+			try {
+				Connection connection = JDBCDriver.getConnection(
+					jdbcURL,
+					new Properties() {
+						{
+							put("password", "");
+							put("user", "sa");
+						}
+					});
```

Comment:
@thienquach87 Can we guarantee that these properties will not change?

Reply:
I updated source